.travis.yml

```diff
@@ -25,6 +25,8 @@ script:
   - export BOTO_CONFIG=/dev/null
   - make trufflehog
   - make test
+  - make check_clean
```

Comment:
Consider moving these before `make test` since it will give quicker feedback in the event of a failure.

Reply:
The main intent behind `check_clean` is to detect tests that dirty the working copy. So that's why it follows `test`. A failing test is a stronger signal about the quality of a PR than lack of squashability. I wouldn't want to mask a test failure with behind non-squashability. I don't usually even look at PRs with failing tests. I might review one that's not squashable.